jetson/cameras/src/__main__.py

```diff
@@ -6,16 +6,27 @@
 import jetson.utils  # noqa
 
 __lcm: lcm.LCM
-__pipelines = [None] * 2
+__pipelines = [None] * 4
 
-ARGUMENTS = ['--headless', '--bitrate=300000', '--width=256', '--height=144']  # Change to desired bitrate e.g. 2000000
-remote_ip = ["10.0.0.1:5000", "10.0.0.1:5001"]
+ARGUMENTS_LOW = ['--headless', '--bitrate=300000', '--width=256', '--height=144']
+ARGUMENTS_MEDIUM = ['--headless', '--bitrate=300000', '--width=1280', '--height=720']
+remote_ip = ["10.0.0.1:5000", "10.0.0.1:5001", "10.0.0.2:5000", "10.0.0.2:5001"]
```

Comment:
can you add a comment explaining both remote ips? (like science laptop vs base station)
also do we not want to be able to stream 4 during ES or smth now?

Reply:
yea.
and i'll go talk with cam about changing code + lcms to stream 4 during ES/maybe others

Reply:
are these resolutions? should the higher resolution also have a higher bitrate?

Reply:
Uh the higher resolution doesn't need a higher bitrate, i think all it does is change the color slightly? I'm not too sure but the fps looks the same and the resolution looks better and the bit rate is still the same so I didn't bother changing it

Reply:
does higher resolution not take up more bandwidth then?

Reply:
It doesn't take more bandwidth if you put bitrate = 300,000

Reply:
ok nvm i take back what i said. u can change the bandwidth to a certain extent until it just ignores the bitrate (e.g. i can keep resolution at 256 by 144 and then change bitrate between the numbers 100,000 and 1,000,000 and it acts normally, but if u test other numbers such as 50,000 it doesn't act properly. if we want a high quality option such as 720p, the lowest bitrate can be is 1,800,000 bps. Lowest for 360p is 800,000 bps. not sure if we want to have any of these options so for now, i'll return it back to what we currently have (256 x 144 at 300,000 bps)